packages/cli/__tests__/zosfiles/__unit__/upload/ftds/__snapshots__/FileToDataSet.handler.unit.test.ts.snap

```diff
@@ -1,18 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Upload file-to-data-set handler process method should display error when upload file to data set 1`] = `
-Object {
-  "apiResponse": Array [
-    Object {
-      "from": "test-file",
-      "success": false,
-      "to": "testing",
-    },
-  ],
-  "commandResponse": "uploaded",
-  "success": false,
-}
-`;
+exports[`Upload file-to-data-set handler process method should display error when upload file to data set 1`] = `undefined`;
 
 exports[`Upload file-to-data-set handler process method should display error when upload file to data set 2`] = `""`;
```

Comment:
not something to resolve on this PR, but I'm curious if we can avoid the use of snapshots for empty strings or undefined/null values 😋